### PR TITLE
Veto definer file should have a PFN

### PR DESCRIPTION
**Pull request by duncan-brown**
The veto definer file is passed to ```pycbc_page_vetotable``` as a bare string. This fixes it so that it is a ```File()``` object and appears correctly in the DAX.

### CODE REVIEW

**Comment by duncan-brown** [comment]
Test running at https://sugar-dev2.phy.syr.edu/pegasus/u/dbrown/r/167/w?wf_uuid=feedff07-2063-4ad5-8c3c-e29c3ac3cb5b

File: pycbc/workflow/plotting.py [hunk 1]
```diff
@@ -207,11 +208,19 @@ def make_veto_table(workflow, out_dir, vetodef_file=None, tags=None):
     if vetodef_file is None:
         vetodef_file = workflow.cp.get_opt_tags("workflow-segments",
                                            "segments-veto-definer-file", [])
+
```
Comment (ahnitz) [inline pycbc/workflow/plotting.py]: These lines should be within the if statement that pulls it from the workflow. It should be a File object if coming from elsewhere.

**Comment by duncan-brown** [comment]
@ahnitz thanks, fixed. Testing now.

**Comment by duncan-brown** [comment]
Confirmed that this does the right thing:
```xml
<file name="H1L1-HOFT_C00_O1_CBC.xml">
      <pfn url="file:///home/dbrown/projects/osg/sugar-nonsharedfs-test-12/output/results/1._analysis_time/1.01_segment_data/H1L1-HOFT_C00_O1_CBC.xml" site="local"/>
</file>
```

**Pull request merged**